TLSbo90665_support12K_playback.patch

Added support for 12K sample rate playback on stereo dac
===================================================================
diff -uNpr linux-2.6.19.2.orig/sound/arm/mxc-alsa-pmic.c linux-2.6.19.2/sound/arm/mxc-alsa-pmic.c
--- linux-2.6.19.2.orig/sound/arm/mxc-alsa-pmic.c	2007-04-25 17:37:36.000000000 -0600
+++ linux-2.6.19.2/sound/arm/mxc-alsa-pmic.c	2007-04-25 17:37:40.000000000 -0600
@@ -711,6 +711,10 @@ void normalize_speed_for_pmic(snd_pcm_su
 			pmic_device->sample_rate = STDAC_RATE_11_025_KHZ;
 			break;
 
+		case 12000:
+			pmic_device->sample_rate = STDAC_RATE_12_KHZ;
+			break;
+
 		case 16000:
 			pmic_device->sample_rate = STDAC_RATE_16_KHZ;
 			break;
